app/AndroidManifest.xml

```diff
@@ -6,7 +6,9 @@
           android:versionName="2.26">
 
     <uses-sdk
-        tools:overrideLibrary="com.etsy.android.grid"
+        tools:overrideLibrary="com.etsy.android.grid, com.google.android.gms.measurement,
```

Comment:
Does anyone know the implications of using this? Will it just fail gracefully on versions that don't support the library, or do something more sinister?